src/taipy/core/task/task.py

```diff
@@ -192,3 +196,49 @@ def submit(
         from ._task_manager_factory import _TaskManagerFactory
 
         _TaskManagerFactory._build_manager()._submit(self, callbacks, force, wait, timeout)
+
+    @classmethod
+    def _to_model(cls, task) -> _TaskModel:
```

Comment:
```suggestion
    def _to_model(cls, task: Task) -> _TaskModel:
```

Reply:
@trgiangdo 
For some reason, type hint doesn't work for `_to_model` and `_from_model` for Task and Datanode, it does for scenario and pipeline. pre-commit is also complaining non-consistent between the 2 functions in _Entity class and actual entity classes. So I decided it's not worth the pain 😃